cmd/edit/ingress/cmd.go

```diff
@@ -174,6 +179,15 @@ func init() {
 		"Components route TLS secret reference for oauth, console, download.",
 	)
 
+	flags.BoolVar(
+		&args.bypassLegacyIngressSupportVersionValidation,
+		bypassLegacyIngressSupportVersionValidationFlag,
+		false,
```

Comment:
Allows customers that opened the support case to move their 4.13 clusters to be under the managed ingress improvements support.